src/WebJobs.Script/Workers/SharedMemoryDataTransfer/MemoryMappedFileAccessor.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.IO.MemoryMappedFiles;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Script.Workers.SharedMemoryDataTransfer
+{
+    /// <summary>
+    /// Encapsulates functionality for accessing <see cref="MemoryMappedFile"/>.
+    /// </summary>
```

Comment:
We probably want to mention that there are differences between Windows and Linux that are handled here.
What about adding comments to the methods too?

Reply:
Method comments are inherited from the interface `IMemoryMappedFileAccessor`.